discovery/gossiper.go

```diff
@@ -1218,6 +1218,14 @@ func (d *AuthenticatedGossiper) retransmitStaleAnns(now time.Time) error {
 		// introduction of the MaxHTLC field, then we'll update this
 		// edge to propagate this information in the network.
 		if !edge.MessageFlags.HasMaxHtlc() {
```

Comment:
It never gets here for private channels, so the migration isn't always performed at this point.

Reply:
Removed migration changes from the pr